js/course_settings_disable_course_conclude.js

```diff
@@ -52,6 +52,8 @@ function initHUGlobal() {
         disableCourseUnconcludeButton();
         addCourseUnconcludeButtonDisabledMessage();
     }
+    
+    addCourseConcludeDateDisabledMessage();
```

Comment:
Shouldn't the message be displayed only upon the datepicker being disabled? Trying to understand why addCourseConcludeDateDisabledMessage can't happen on line 65 as before?